### ISSUE TITLE
RFE: Use the ansible reboot module instead of own code

### ISSUE THREAD

**Issue by Tronde** [Tronde/ansible-role-rhel-patchmanagement#10]
In tasks/main.yml the following code is used to reboot hosts that have installed any kind of update:

~~~
 - name: Reboot Host if packages were updated
    shell: sleep 2 && shutdown -r now "Ansible updates triggered"
    async: 1
    poll: 0
    ignore_errors: true
    when: ("Complete!" in yum_output.stdout_lines[-1]) or
          ("Komplett!" in yum_output.stdout_lines[-1])
~~~

Instead of the shell module the [ansible module reboot](https://docs.ansible.com/ansible/2.9/modules/reboot_module.html) could be used here.

I don't know yet, when I'll be able to implement and test this RFE. In case anyone would like to do this, you are welcome to contribute.

**Comment by Tronde** [comment]
Changes are going to be tested on 2022-01-11.
When everything looks good I move the changes to `dev` branch and merge with `master`.

**Comment by Tronde** [comment]
Closed with #13.
